src/components/entries/NumberField.js

```diff
@@ -58,7 +58,7 @@ export function NumberField(props) {
 
   return (
     <div class="bio-properties-panel-numberfield">
-      {displayLabel && <label for={ prefixId(id) } class="bio-properties-panel-label">{ label }</label> }
+      {displayLabel && <label htmlFor={ prefixId(id) } class="bio-properties-panel-label">{ label }</label> }
```

Comment:
This change is coming from React limitations. I'd rather adjust the rules to not enforce it as we are fine in preact world with such attributes.